### PR TITLE
docs: add data layer schema

### LINKED ISSUE #331: [Doc][Data Models] Add a doc to explain the three-layer data models
## Documentation Scope
Add a doc to explain the three-layer data models under the [data models](https://devlake.apache.org/docs/next/DataModels) folder

## Describe the Change
To give the user an overview of the data layers stored in DevLake MySQL. Now there're only domain layer models, tool layer and raw layer data models are not introduced to the users.

## Screenshots
image

### PR DESCRIPTION
**Pull request by Startrekzky**
### ⚠️ &nbsp;&nbsp;Pre Checklist

> Please complete _ALL_ items in this checklist, and remove before submitting

- [x] I have `npm run build` and `npm run serve` locally before submitting this PR
- [x] I have read through the [Contributing](https://devlake.apache.org/community/) Documentation

# Summary
- add raw layer schema
- add tool layer schema
- add system tables
- update domain layer schema

### Does this close any open issues?
Closes #331 

### Screenshots
image

### CODE REVIEW

File: docs/DataModels/ToolLayerSchema.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+description: >
+   Extract raw data into a relational schema for each specific tool
+sidebar_position: 2
+---
+
+## Summary
+
+This document describes Apache DevLake's tool layer schema.
+
+Referring to DevLake's [architecture](../Overview/Architecture.md), the Tool layer extracts raw data from JSONs into a relational schema that's easier to consume by analytical tasks. Each DevOps tool would have a schema that's tailored to their data structure, hence the name, the Tool layer.
```
Comment (keon94) [inline docs/DataModels/ToolLayerSchema.md]: grammar: "tailored to their data structure" -> "tailored to _its_ data structure"

File: docs/DataModels/SystemTables.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+## Data Models
+
+These tables start with a prefix `_devlake`. Unlike raw or tool data tables, DevLake only contains one set of system tables. The naming convension of these tables is `_raw_{plugin}_{entity}`, such as 
+- _devlake_blueprints
+- _devlake_pipelines
+- _devlake_tasks
+- _devlake_subtasks
+- ...
+
+Normally, you do not need to use these tables, unless you have use cases above.
```
Comment (keon94) [inline docs/DataModels/SystemTables.md]: "unless you have use cases above" can be worded a bit better: "unless you have one of the above use cases"

File: docs/DataModels/SystemTables.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+## Use Cases
+
+1. As a user, you can check `_devlake_blueprints` and `_devlake_pipelines` when failing to collect data via DevLake's blueprint.
+2. As a contributor, you can check these tables to debug task concurrency or data migration features.
+
+
+## Data Models
+
+These tables start with a prefix `_devlake`. Unlike raw or tool data tables, DevLake only contains one set of system tables. The naming convension of these tables is `_raw_{plugin}_{entity}`, such as
```
Comment (keon94) [inline docs/DataModels/SystemTables.md:21]: Let's add "These tables are used and managed by the Devlake framework."

**Comment by keon94** [comment]
LGTM

**Review by warren830 — APPROVED** [review]
LGTM

**Pull request merged**